ui/src/sentry.ts

```diff
@@ -0,0 +1,26 @@
+import * as Sentry from '@sentry/react'
+import { Integrations } from '@sentry/tracing'
+
+import { initSentryRoutingInstrument } from '@lib/utils/sentryHelpers'
+
+const version = process.env.REACT_APP_VERSION
+const name = process.env.REACT_APP_NAME
+const hash = process.env.REACT_APP_COMMIT_HASH
```

Comment:
Maybe better to use the information in Dashboard Version page? I'm just not sure in what case these two information will be mismatched.

Reply:
@breeswish 

It might be better to have this kind of data during build time.

There are some version infos can be retrieved from `/info/info` API, but then we'll have to delay the sentry init until API response arrives if we relies on that. Also the API/network might fail or takes longer time than we expected, so...